files/es/web/api/batterymanager/dischargingtime/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: BatteryManager.dischargingTime
+slug: Web/API/BatteryManager/dischargingTime
+tags:
+  - API
+  - Apps
+  - Batería
+  - Battery
+  - Firefox OS
+  - Gecko DOM Reference
+  - WebAPI
+  - aplicaciones
+translation_of: Web/API/BatteryManager/dischargingTime
+browser-compat: api.BatteryManager.dischargingTime
+---
+
+{{deprecated_header}}{{APIRef("Battery API")}}
+
+Indíca la cantidad de tiempo, en segundos,
+que restan antes de que la batería se descargue completamente.
+
+> **Nota:** Incluso si el tiempo devuelto es exacto al segundo,
+> los navegadores lo redondean a valor más alto.
+> intervalo (típicamente a los 15 minutos más cercanos) por razones de privacidad.
+
+## Sintaxis
+
+```js
+var time = battery.dischargingTime;
+```
+
+Valor de retorno, `time` es el tiempo restante en segundos antes que la `batería`,
+que es un objeto {{domxref("BatteryManager")}},
+esté completamente descargada y el sistema se suspenda.
+Este valor es [`Infinito`](/es/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+Si la batería se está cargando en vez de descargando,
+o si el sistema es incapaz de calcular el tiempo de descarga.
+
+## Ejemplo
+
+### Contenido HTML
+
+```html
+<div id="dischargingTime">(tiempo de descarga desconocido)</div>
+```
+
+### Contenido JavaScript
+
+```js
+navigator.getBattery().then(function (battery) {
+  var time = battery.dischargingTime;
+
+  document.querySelector("#dischargingTime").textContent = battery.dischargingTime;
+});
+```
+
+{{ EmbedLiveSample('Ejemplo', '100%', 30) }}
```

Comment:
This macro can be a bit tricky/"fragile" for the translated-content side. Right now, it's "safer" to **pass an empty string as the first argument to `{{EmbedLiveSample}}`s** (see [this issue at mdn/yari](https://github.com/mdn/yari/issues/4971) for more details):

```suggestion
{{EmbedLiveSample('', '100%', 30)}}
```

N.B. I see that the en-US file has spaces between `{{` and the macro contents/arguments, which I'd say should be avoided, so this is something I'd correct on the translated side even if the English version uses the not-so-proper styling/format.

Reply:
if I leave it as a blank value it doesn't work, it shows an empty rectangle instead of the `Infinity` word.
![image](https://user-images.githubusercontent.com/7331511/152282698-c3a4bd15-8664-4cd0-9185-dac8d0dfe8d5.png)

Reply:
Ah, never mind this suggestion for this file then :) 

I think I figured out why — in most files (especially docs for non-deprecated features), the `{{EmbedLiveSample}}` macro is always under its own "Result" heading. 

For reference, I tested it with this and was able to render the sample without a problem:

```
...
document.querySelector("#dischargingTime").textContent = battery.dischargingTime;
});
...

### Resultado

{{EmbedLiveSample('', '100%', 30)}}
```

<details>
<summary>See the image here:</summary>

![dischargingtime-sample-macro](https://user-images.githubusercontent.com/87150472/152284223-d59c82d2-6048-4d08-a951-202b6b9e2528.png)

</details>

However, since there is no such Results heading for this page, it actually needs the argument. Thanks for bringing this to my attention!

Reply:
wow! I'm learning a lot with your help, thank you for being here @cw118  :partying_face: 

I think I'll add the ### Resultado subtitle even if it is not in the English version,
it makes it easy to read and understand, what do you think?

Reply:
I'm learning so much as well :)

I'll leave the decision (whether to add that heading) to you as a reviewer since it's not in any other locale; I don't think it'd be a problem and I do agree about it being more readable.

Reply:
cool! I added it